Dockerfile

```diff
@@ -53,7 +102,16 @@ RUN microdnf update -y && \
         libgomp \
         mesa-libGL \
         glib2-devel \
-        shadow-utils
+        shadow-utils && \
+        if [[ "${OPT_MLSERVER_SYS_TRACING}" == "sdt-sys-tracing" ]]; then \
+            microdnf install -y \
+                elfutils-libelf; \
+        fi
```

Comment:
Would there be any trade-off to always installing it on the image? 

Mainly thinking that, if we want to make this an opt-in feature within MLServer, then it makes sense it's always possible to enable it on the official set of images. Otherwise, we would need to build a second set of images which have these installed.

Reply:
The main reason I wanted to offer this as an option is in case there are users who see `libelf` and `libstapsdt` as increasing their attack surface in the container. So perhaps all images provided by Seldon would have those installed, but users building their own have the ability of not adding them in.

If that represents just a couple of users requiring extreme security, perhaps the way forward would be to just offer a simple way of removing those from an existing image rather than installing extras conditionally.

Reply:
Got it - I'd then just remove the option from the `Dockerfile` and always install those. This simplifies maintenance of the Dockerfile, and if we need to we can always add this back later on.